File: TUF/Numbers/ReverseDigitsOfNumber.java
import java.util.Scanner;

public class ReverseDigitsOfNumber {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = input.nextInt();
        System.out.println(num + " reverse is: " + reverseNumber(num));
        input.close();
    }

    private static int reverseNumber(int num) {
        if (num <= 10) {
            return num;
        } else {
            int revNum = 0;
            while (num != 0) {
                int rem = num % 10;
                revNum = revNum * 10 + rem;
                num = num / 10;
            }
            return revNum;
        }
    }
}

File: TUF/Numbers/ReverseDigitsOfNumber.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function reverseNumber(num) {
    if (num <= 10) {
        return num;
    }
    else {
        var revNum = 0;
        while (num !== 0) {
            var rem = num % 10;
            revNum = revNum * 10 + rem;
            num = Math.floor(num / 10);
        }
        return revNum;
    }
}
var num = 45678;
console.log("Reverse of: ".concat(num, " is: ").concat(reverseNumber(num)));
